src/browser/of_events.ts

```diff
@@ -42,6 +42,15 @@ class OFEvents extends EventEmitter {
 
         return super.emit(route, ...data);
     }
+
+    public SUBSCRIBER: StringMap = {
```

Comment:
Same here, static? Also, why uppercase, I mean, I kinda get it why, but then again... "why"?